policy-report/api/v1alpha1/policyreport_types.go

```diff
@@ -48,7 +48,7 @@ type PolicyReportResult struct {
 
 	// Resource is an optional reference to the resource check bu the policy rule
 	// +optional
-	Resource *corev1.ObjectReference `json:"resource,omitempty"`
+	Resource []*ResourceDefination `json:"resource,omitempty"`
```

Comment:
Do we need a GVK for the resources? Seems like that would not support heterogeneous resources....

Can we simply change to `Resources *[]corev1.ObjectReference` to allow for multiple resources?